app/controllers/rooms/room_tickets_controller.rb

```diff
@@ -0,0 +1,49 @@
+class Rooms::RoomTicketsController < ApplicationController
+  before_action :auth_user
+  before_action :set_room_ticket, only: %i[ show ]
+  before_action :set_room, only: %i[ new send_email_for_tdx_ticket ]
+
+  # GET /room_tickets or /room_tickets.json
+  def index
+    @room_tickets = RoomTicket.all
+    authorize @room_tickets
+  end
+
+  # GET /room_tickets/new
+  def new
+    @room_ticket = RoomTicket.new
+    authorize @room_ticket
+  end
+
+  def send_email_for_tdx_ticket
+    date = Time.now.strftime('%m/%d/%Y')
```

Comment:
No need to create a date variable and send it to the mailer. In the mailer, you can use DateTime.now See me comment for the mailer.